mongo/errors.go

```diff
@@ -612,6 +612,56 @@ func (bwe BulkWriteException) HasErrorCodeWithMessage(code int, message string)
 // serverError implements the ServerError interface.
 func (bwe BulkWriteException) serverError() {}
 
+// ClientBulkWriteException is the error type returned by ClientBulkWrite operations.
+type ClientBulkWriteException struct {
+	// A top-level error that occurred when attempting to communicate with the server
+	// or execute the bulk write. This value may not be populated if the exception was
+	// thrown due to errors occurring on individual writes.
+	TopLevelError *WriteError
```

Comment:
Cannot use Error as a field name because of the conflict with the conventional method name.

Reply:
Optional: Consider the more conventional `Err` or `WriteError`.